source/outlines/2016-03-09.markdown

```diff
@@ -0,0 +1,123 @@
+---
+title: 20160309
+layout: outline
+---
+
+## All
+
+### Word of the Day
+
+* invective - noun - an abusive expression or speech; a vehement verbal
+attack
+
+
+## 1508
+
+### Class (9:00 - 12:00)
+
+### Final Assessment Evaluations (1:00 - 4:00)
+
+#### With Meeka
```

Comment:
I would say maybe 10 people will actually need to get a final assessment - so we definitely won't need all of Tess/Lovisa/Jorge